external-crates/move/crates/move-compiler/src/shared/mod.rs

```diff
@@ -866,3 +868,124 @@ macro_rules! process_binops {
 }
 
 pub(crate) use process_binops;
+
+//**************************************************************************************************
+// Virtual file system support
+//**************************************************************************************************
+
+/// Same as `IndexedPackagePath` but the path is a virtual file system added/
+/// symbol.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct IndexedVfsPackagePath {
```

Comment:
Is it worth just making `IndexedPackagePath` generic over the path type?

Reply:
Like we could have 
```
pub struct IndexedPackagePathSOME_SUFFIX_THAT_IS_MEANINGFUL<P> { ... }
pub type IndexedPackagePath = IndexedPackagePathSOME_SUFFIX_THAT_IS_MEANINGFUL<Symbol>;
pub type IndexedVfsPackagePath = IndexedPackagePathSOME_SUFFIX_THAT_IS_MEANINGFUL<VfsPath>
```

Reply:
Not a big deal, just might help long term

Reply:
Good idea, but rather than figuring out a meaningful suffix (which I couldn't...), I kept `IndexedPackagePath` as the struct name and used `IndexedVfsPackagePath` and `IndexedPhysicalPackagePath` for actual types (was thinking about simply `IndexedFsPackagePath` for the latter but this would arguably make the two types look too similar).